staging/src/k8s.io/library-go/scheduling/corev1helpers/BUILD

```diff
@@ -0,0 +1,34 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
Is it possible to rename `library-go/scheduling/corev1helpers` to simply `library-go/scheduling/helpers`? 
IMHO, no matter what the group version is, they are all helpers.

Reply:
Not against that. Right now, the idea is to put here all the helpers that are needed by the scheduling framework and at least other separate component/binary. Though, the version might still help to minimize indirect dependencies when importing `k8s.io/library-go/scheduling/NAME` package. On the other hand we might omit `helpers` in the package name and have `k8s.io/library-go/scheduling/core/v1` instead? Assuming the staging repo gets renamed from `library-go` to e.g. `helpers`.

Reply:
Going with `k8s.io/component-helpers/scheduling/corev1`. Though, the topic is still open for discussion.

Reply:
I am OK with this naming, it looks much better.